src/components/KSelect/KSelectItems.vue

```diff
@@ -59,7 +63,87 @@ const emit = defineEmits<{
   (e: 'selected', item: SelectItem): void
 }>()
 
-const handleItemSelect = (item: SelectItem) => emit('selected', item)
+const handleItemSelect = (item: SelectItem) => {
+  return emit('selected', item)
+}
+
+// Ref to store focusable items
+const focusableItems = ref<HTMLElement[]>([])
+
+// Function to handle the keydown event
+const handleKeyDown = (event: KeyboardEvent, item: SelectItem) => {
```

Comment:
(not sure where to place this comment)
Pressing "Enter" to select an element does not close the dropdown

Reply:
Can you recheck? I have tried both Space/Enter key and it closes as expected